app/src/ui/lib/branch-name-warnings.tsx

```diff
@@ -46,3 +46,26 @@ export function renderBranchHasRemoteWarning(branch: Branch) {
     return null
   }
 }
+
+export function renderBranchNameExistsOnRemoteWarning(
+  sanitizedName: string,
+  branches: ReadonlyArray<Branch>
+) {
+  const alreadyExistsOnRemote =
+    branches.findIndex(
+      b => b.nameWithoutRemote === sanitizedName && b.type === BranchType.Remote
+    ) > -1
+
+  if (alreadyExistsOnRemote) {
```

Comment:
:art: invert this check so you can return early

```tsx
if (!alreadyExistsOnRemote) {
  return null
}

return (
  ...
)
```

Reply:
Sounds good, I will make this change now. 

There is a second function in this file, `renderBranchHasRemoteWarning`, that also follows the if(...) {return warning} else { return null } structure. Would you like me to modify that to conform with this change as well?